README.md

```diff
@@ -1,2 +1,181 @@
-# ReactCapstone
-/
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+   - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [WORLD EVENTS] <a name="about-project"></a>
+
+> "World Events" is a mobile application that provides users with comprehensive details about various events happening around the world. Whether it's concerts, sports matches, theater performances, conferences, or any other kind of event, World Events serves as a one-stop platform for discovering, exploring, and participating in exciting happenings worldwide.
+
+## 🛠 Built With <a name="React"></a>
+
+1. React
+2. Redux
+3. HTML
+4. CSS
+5. JAVASCRIPT
+6. LINTERS
+7. ES6
+8. GitFlow
+
+### Tech Stack <a name="Front end"></a>
+
+> This is a mobile browser website.
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://react.dev/">React</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+### Key Features 
+
+1. Create events page
+2. Create event details page
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="https://worldevents.netlify.app/"></a>
+
+
+- [[Live Demo Link](https://drive.google.com/file/d/1Ffo2oY4h1P_3chOLN-yDSQJlsh9vq10G/view?usp=sharing)]
```

Comment:
- [ ] As mentioned by the previous reviewer in the `Live demo` section you should put the link of the live demo "deployed link" and create another section for the video presentation and put the video link. Kindly fix that 🙏🏾 

```
[[Live Demo Link](https://worldevents.netlify.app/)]

[[Video Presentation Link](https://drive.google.com/file/d/1Ffo2oY4h1P_3chOLN-yDSQJlsh9vq10G/view?usp=sharing)]
```

Reply:
Thank you